src/main/webapp/WEB-INF/views/admin/selfCheck/edit.jsp

```diff
@@ -141,9 +141,9 @@
 		<div class="btnLayout">
 			<input id="delete" type="button" value="Delete" class="btnColor left selfCheckDelete" /><!-- 2018-07-19 choye 추가 class에  selfCheckDelete -->
 			<span class="right">
+				<input type="button" value="OSS Notice" onclick="createTabInFrame('Notice','#/selfCheck/verification/'+'${project.prjId}')" class="btnColor red btnNotice srcBtn"/>
```

Comment:
I think you need to increase the button width

![image](https://user-images.githubusercontent.com/32615702/143342111-ae6164f2-de53-41d2-a68a-47f643fd60f8.png)

Reply:
Fix it like below!
![image](https://user-images.githubusercontent.com/53862866/143386853-b4dc623b-6cdb-47f5-a4d0-6447aa089e34.png)

Reply:
LGTM :+1: